### PR TITLE
Improved design of the repository

### PR DESCRIPTION
**Pull request by StefanDjokovic**
After many iteration I found this to be a great design for the ReadMe of this repository. This uses a very minimalistic approach of plan markdown text, while adding the contribution of emojis that increases engagement of the users.

### CODE REVIEW

**Comment by Giuzzilla** [comment]
I agree with the idea and overall direction, but haven't reviewed the code changes or tested them.

Can you please include in the PR tests reflecting the features that you are adding?

**Comment by Alexdruso** [comment]
> 
> 
> I agree with the idea and overall direction, but haven't reviewed the code changes or tested them.
> 
> Can you please include in the PR tests reflecting the features that you are adding?

I do agree with @Giuzzilla and I would also like to add that, from my experience, this pr is lacking some kek factor. Could  you increase it?

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# advent-of-code-2020
\ No newline at end of file
+# 🎅 My Advent of Code 2020 Solutions 🧑‍🎄
```
Comment (Alexdruso) [inline README.md:1]:
```suggestion
# 🎅 My Advent of Code 2020 Kek Solutions 🧑‍🎄
```

Comment (StefanDjokovic) [reply README.md:1]: Hi @Alexdruso , great edit! Love it, keep it up!
If you are interested I have a start-up that sells kek factors solutions. We are looking for a Cobol developer that would help us adding keks into banks. Would you be interested?

**Comment by StefanDjokovic** [comment]
> I agree with the idea and overall direction, but haven't reviewed the code changes or tested them.
> 
> Can you please include in the PR tests reflecting the features that you are adding?

Hi @Giuzzilla , it works on my machine, so I'm quite confident that it would work also on any machine.

**Comment by Giuzzilla** [comment]
Plugged it into an IntCode machine and I can assure you the results are NOT pretty.

**Comment by StefanDjokovic** [comment]
> Plugged it into an IntCode machine and I can assure you the results are NOT pretty.

@Giuzzilla It works perfectly on my, @davidemerli 's and @Danis98 . Consider fixing your bugged IntCode machine.

File: README.md (hunk 1 above)
Comment (Alexdruso) [reply README.md:1]: I have always been into Finkek! Sure, I want to be working on this new exciting kektional language!

**Review by S0NN1 — APPROVED** [review]

**Pull request merged**